prowler/providers/aws/services/ec2/ec2_ebs_block_public_access_snapshots/ec2_ebs_block_public_access_snapshots.py

```diff
@@ -0,0 +1,30 @@
+from prowler.lib.check.models import Check, Check_Report_AWS
+from prowler.providers.aws.services.ec2.ec2_client import ec2_client
+
+
+class ec2_ebs_block_public_access_snapshots(Check):
+    def execute(self):
+        findings = []
+        for (
+            ebs_snapchot_block_status
+        ) in ec2_client.ebs_block_public_access_snapshots_state:
+            if (
+                ebs_snapchot_block_status.snapshots
+                or ec2_client.provider.scan_unused_services
+            ):
+                report = Check_Report_AWS(self.metadata())
+                report.region = ebs_snapchot_block_status.region
+                report.resource_arn = f"arn:aws:ec2:{ebs_snapchot_block_status.region}:{ec2_client.audited_account}"
```

Comment:
I think you can retrieve this from the ec2 client.

Reply:
Yes, please do it the same as `s3_account_level_public_access_blocks` check